colabs/Wczytanie_danych.ipynb

```diff
@@ -846,6 +894,922 @@
           "execution_count": 13
         }
       ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Załadowanie nowych baz\n",
```

Comment:
Takie setupy powinny być na czele notatnika.

Reply:
To nie moje zadanie - to zadanie Emilki, zmiany pojawiają się również tutaj, bo request jest na merge do maina

Reply:
potraktować jak dodatkowy refactor.